src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 #include <map>
 #include <set>
 #include "JasmineGraphFrontEnd.h"
+#include <nlohmann/json.hpp>
```

Comment:
This is a new library for JasmineGraph. We need to know the license and also update the information in the README.

Reply:
This has [MIT license](https://github.com/nlohmann/json/blob/develop/LICENSE.MIT), Will update the README file as well

Reply:
Fixed in https://github.com/miyurud/jasminegraph/pull/122/commits/bdd734e88390169ddc9e44c74030d1c1b170d0ae

Reply:
Ok